recipes/default.rb

```diff
@@ -104,6 +104,9 @@
     :dir          => node["collectd"]["dir"],
     :interval     => node["collectd"]["interval"],
     :read_threads => node["collectd"]["read_threads"],
+    WriteQueueLimitHigh: node['collectd']['WriteQueueLimitHigh'],
```

Comment:
Same here. Can you please make these consistent with the other attributes?

- Single quotes to double quotes
- Snake case names
- Old style symbols